src/test/unit_tests/test_nb_native_hashes.js

```diff
@@ -36,8 +40,8 @@ mocha.describe('nb_native hashes', function() {
             '123456789',
             '1234567890',
         ]) {
-        mocha.it(`MD5 ${s}`, function() {
-            md5(Buffer.from(s));
+        mocha.it(`MD5 ${s}`, async function() {
+            await md5(Buffer.from(s));
```

Comment:
Why do you replace the other md5 call and not just add your calls?
We want the test to keep validating both impls for now.

Reply:
It is validating both implementations, I've just transformed this to async and called mine inside as well.
I can add another function for mine that does the same thing if that is what you've meant.